plugins/offboard/offboard_impl.cpp

```diff
@@ -257,6 +284,55 @@ void OffboardImpl::send_velocity_body()
     _parent->send_message(message);
 }
 
+void OffboardImpl::send_altitude()
+{
+    const static uint16_t IGNORE_X			= true	<< 0;
+    const static uint16_t IGNORE_Y			= true	<< 1;
+    const static uint16_t IGNORE_Z			= false	<< 2; // No ignore for altitude
```

Comment:
I find it weird to shift a boolean. It probably works but `1` is probably more common.